script.js

```diff
@@ -1,139 +1,249 @@
 
-
+//exercise to translate words
+//VARIABLES
+// counter trough items(words)
 let countItem =0;
+// id input from user in inputfield 
 let inputValue = document.getElementById("inputvalue");
+//id button to skip or to go to the next item 
 const submitButton = document.getElementById("submitButton");
+//lesson number for progress bar 
+let lessonNumber=0;
+//id from progress bar
+let progress= document.getElementById("progress");
+// points for having good or wrong awnsers
+let score=0;
+// id for score in banner
+let idScore= document.getElementById("score");
```

Comment:
you are not getting the id of the score but a reference to the element having id='score'